### ISSUE TITLE
Failing test: tests/plugins/autocomplete/manual/a11ydivarea (mobile)

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4#4712]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

1. Run teset `tests/plugins/autocomplete/manual/a11ydivarea`

### Expected result

This test should be ignored on mobile

### Actual result

The test is available.

## Other details

* Browser: Chrome (mobile) 90
* OS: Android 10
* CKEditor version: 4.16.1
* Installed CKEditor plugins: -

**Comment by f1ames** [comment]
Fixed in #4724.
